sycl/include/CL/sycl/usm.hpp

```diff
@@ -20,133 +20,133 @@ namespace sycl {
 ///
 __SYCL_EXPORT void *malloc_device(
     size_t size, const device &dev, const context &ctxt,
-    const detail::code_location CL = detail::code_location::current());
+    const detail::code_location codeLoc = detail::code_location::current());
```

Comment:
To follow LLVM Style Guidelines
```suggestion
    const detail::code_location CodeLoc = detail::code_location::current());
```

Reply:
Okay, I'll change it.